components/size_aware_image.jsx

```diff
@@ -126,6 +128,8 @@ export default class SizeAwareImage extends React.PureComponent {
             // Generate a blank image as a placeholder because that will scale down to fit the available space
             // while maintaining the correct aspect ratio
             src = createPlaceholderImage(dimensions.width, dimensions.height);
+        } else if (this.state.error) {
+            return null;
```

Comment:
Doing this will cause a pop in scroll. 
I wonder if we should leave the `createPlaceholderImage` even when it fails to prevent it thoughts?. 
We can show an error message after talking to the UX on top of the placeholder for the next release or something

Reply:
I maybe wrong but the potential scroll pop here is just the same as what we have right now I believe.

When no dimension, it will do (1) `src = this.props.src;` then return (2) `null` on error. On (1), there's no image to show so no scroll pop. If image loaded, there will scroll pop but just the same currently.

When there's dimension, it will (1) render placeholder and loading indicator, then the (2) image itself renders once resolve (so no scroll pop). In case of error, there will be scroll pop but just the same currently.

Reply:
> in case of error, there will be scroll pop but just the same currently.

aah ya, got it. Makes sense we can create another ticket for image error fallback when metadata is enabled.